### PR TITLE
Swapped values() for itervalues() for python3 compatibility.

### PR DESCRIPTION
**Pull request by JivanAmara**
```
ignore_nodes = set()
process_in_wave1 = dep_graph.ind_nodes(ignore_nodes=ignore_nodes)
...
ignore_nodes.update(process_in_wave1)

process_in_wave2 = dep_graph.ind_nodes(ignore_nodes=ignore_nodes)
...
ignore_nodes.update(process_in_wave2)

process_in_wave3 = dep_graph.ind_nodes(ignore_nodes=ignore_nodes)
...
```

### CODE REVIEW

File: dag/__init__.py [hunk 1]
```diff
@@ -144,13 +145,31 @@ def reset_graph(self):
...
+        elif type(ignore_nodes) == set:
+            ignore = ignore_nodes
+
+        # Independent nodes
+        inodes = set()
+        for node, downstream_nodes in graph.items():
+            if len(downstream_nodes - ignore) == 0 and node not in ignore:
+                inodes.update(set([node]))
+
+        return inodes
```
Comment (BruceDone) [inline dag/__init__.py]: it will return the end node of the dag ,not the root node

Comment (JivanAmara) [inline dag/__init__.py]: Are you using an edge A->B as A depends on B or A is a dependency of B?

Comment (JivanAmara) [inline dag/__init__.py]: I think some clarification of terminology would help, a "root" node is a term borrowed from tree structures; I'm pretty sure you mean a node with no edge leading away from it & a dag can have multiple nodes like that.

Comment (andrefun) [inline dag/__init__.py]: A new set is created each time the function is called if the ignore_nodes argument isn’t provided in function in_nodes.

**Comment by thieman** [comment]
Closing. Py3 support has been added by #23.

**Pull request closed without merging**